package.json

```diff
@@ -23,7 +23,7 @@
     "lint": "eslint ./**/*.js --fix",
     "prepare-production": "snyk protect",
     "start":
-      "nodemon ./node_modules/serverless/bin/serverless offline start --skipCacheInvalidation",
+      "DEBUG='fcc:*' nodemon ./node_modules/serverless/bin/serverless offline start --skipCacheInvalidation",
```

Comment:
This whole line is not safe for cross systems. Here is an example of the behaviour on Windows (git bash for Windows):

```
$ yarn start
yarn run v1.6.0
$ DEBUG='fcc:*' nodemon ./node_modules/serverless/bin/serverless offline start --skipCacheInvalidation
'DEBUG' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
I would rather use a task runner like gulp to offload the heavy-lifting from here.

Reply:
We can have this later and separate of this PR. Lets just get the package-lock removed and we are good to go.

Reply:
HOLD on... I do not have `nodemon` globally on a separate *NIX system and I am getting this:

```
❯ yarn start
yarn run v1.6.0
(node:81582) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
$ DEBUG='fcc:*' nodemon ./node_modules/serverless/bin/serverless offline start --skipCacheInvalidation
/bin/sh: nodemon: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

on macOS

I think this is not expected right?

Reply:
@raisedadead scratched his own itch and raised https://github.com/freeCodeCamp/open-api/pull/78

Reply:
Nodemon was installed as a dep. I think the rebase after the switch to yarn messed things up. Will fix

Reply:
Well I must admit that OCD with DX will be a thing around here! 🤷‍♂️

Reply:
Love it.